packages/commercetools/api-client/src/api/getCategory/defaultQuery.ts

```diff
@@ -3,15 +3,15 @@ import gql from 'graphql-tag';
 export default gql`
   fragment Children on Category {
     id
-    slug(locale: $locale)
-    name(locale: $locale)
+    slug(acceptLanguage: $acceptLanguage)
+    name(acceptLanguage: $acceptLanguage)
     childCount
   }
 
   fragment DefaultCategory on Category {
     id
-    slug(locale: $locale)
-    name(locale: $locale)
+    slug(acceptLanguage: $acceptLanguage)
```

Comment:
does it work? param name is `locale` as I remember?

Reply:
slug is a localised field which can take either locale or acceptLanguage ```slug(locale: LocaleacceptLanguage: [Locale!]): String```, difference being singular vs accepts a list of fallbacks